clarifai/client/model.py

```diff
@@ -576,3 +576,179 @@ def __str__(self):
         if hasattr(self.model_info, param)
     ]
     return f"Model Details: \n{', '.join(attribute_strings)}\n"
+
+  def list_evaluations(self) -> List[resources_pb2.EvalMetrics]:
+    """List all eval_metrics of current model version
+
+    Raises:
+        Exception: Failed to call API
+
+    Returns:
+        List[resources_pb2.EvalMetrics]: list of eval_metrics
+    """
+    request = service_pb2.ListModelVersionEvaluationsRequest(
+        user_app_id=self.user_app_id,
+        model_id=self.id,
+        model_version_id=self.model_info.model_version.id)
```

Comment:
Can we have a check and run model.load_info() if version id is none.. Usually model version ID will not be there..

Reply:
You're right, I should check it. I believe I can use an assert statement to check if model_version.id is None?